utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pln1.cpp

```diff
@@ -94,9 +94,9 @@ int main(int argc, char **argv)
     unsigned int outputFormatToggle = atoi(argv[4]);
     int test_case = atoi(argv[5]);
 
-    bool additionalParamCase = (test_case == 8 || test_case == 21 || test_case == 24 || test_case == 40 || test_case == 41 || test_case == 49);
+    bool additionalParamCase = (test_case == 8 || test_case == 21 || test_case == 23 || test_case == 24 || test_case == 40 || test_case == 41 || test_case == 49);
```

Comment:
We have so many file changes for tests which is changed for every new kernel support of RPP. It is very hard to review. We need to take a pause and make combine all these test scrips into single file which can be controlled with arguments. Plus it will increase the productivity of test creation as well since you don't need to repeat this changes to all